internal/compiler/widgets/material-base/widget-spinbox.slint

```diff
@@ -9,26 +9,23 @@ SpinBoxButton := Rectangle {
     property<bool> pressed: self.enabled && touch.pressed;
     property<bool> enabled <=> touch.enabled;
     property<float> icon-opacity: 1;
-    property<brush> icon-fill: md.sys.color.primary;
+    property<brush> icon-fill: md.sys.color.on-primary;
 
     width: height;
 
     container := Rectangle {  
         width: 100%;
         height: 100%;
-        border-radius: 2px;
-        border-width: 1px;
-        background: md.sys.color.secondary-container;
-
-        animate drop-shadow-blur { duration: 250ms; easing: ease; }
+        border-radius: max(width, height) / 2;
+        background: md.sys.color.primary;
     }
 
     state-layer := Rectangle {  
         opacity: 0;
-        width: 100%;
-        height: 100%;
+        width: container.width;
+        height: container.height;
```

Comment:
Ah yes, this is what I thought would be used in `CheckBox`'s `TouchArea` as well :)